lib/gracenote/auth/registration.rb

```diff
@@ -2,12 +2,13 @@ module Gracenote
   module Auth
     module Registration
       def register
-        query("register", client: client_id) do |response|
-          if response.ok?
-            @user_id = response.params["user"]
-          else
-            raise response.message
-          end
+        response = query("register") do |xml|
+          xml.CLIENT client_id
+        end
+        if response.ok?
+          @user_id = response.params["user"]
+        else
+          raise response.message
```

Comment:
__[rubocop]__
* Use `fail` instead of `raise` to signal exceptions. [:ref](https://github.com/bbatsov/ruby-style-guide#fail-method)